### PR TITLE
CV

### PR DESCRIPTION
**Pull request by AlekseyGrimm**
Task: https://github.com/Front-Grodno/Front-Grodno-JS-course-2021/tree/master/modules/css
Deploy: https://alekseygrimm.github.io/HTML-CSS-FE-Course

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" />
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <title>CV</title>
+</head>
+
+<body class="page">
+    <section class="sectionFirst">
```
Comment (YauheniBeiduk) [inline index.html:14]: I think in our projects we need to use BAM methodologhy

File: index.html [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        <img class="foto" src="./img/foto.png" alt="foto">
+        <div class="page_name">
+            <h1 class="name">Alexey Stalpets</h1>
+            <h2 id="blink" class="job">Frontend developer</h2>
+        </div>
+    </section>
+    <div class="container font">
+        <section class="bar indent">
+            <div class="contact">
+                <p><a href="http://#" target="_blank" class="links">
```
Comment (YauheniBeiduk) [inline index.html:24]: Why do you use <p> as like a wrapper ?? I think we can remove it?!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+            <div class="about indent">&#10022;&ensp;ABOUT ME
+                <p>&#8658; I want to join a team that don't know the word stop, in which I can develop.</p>
+                <p>&#8658; I good plan my working hours. I can understand else's developers code, I have an analytical
+                    mind.
+                    Read the book "Stoyan Stefanov - JavaScript Shablony (2011)".
+                    Cheerful and sociable, I always find a common language with the new team.
+                </p>
+            </div>
+            <div class="certificates indent">&#10022;&ensp;CERTIFICATES
+
```
Comment (YauheniBeiduk) [inline index.html]: remove this empty line's ( its general comment)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+                <a href="https://drive.google.com/file/d/1PZfox1HeJsx6TYg_YCvG-ydki-8tXZVY/view?usp=sharing"
+                    target="_blank" div class="certificate">
```
Comment (YauheniBeiduk) [inline index.html:51]: `div` what is it??

File: index.html [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+                    <img src="./img/Certificate-EPAM.png" alt="mail">
+                </a>
+                <a href="https://app.rs.school/certificate/40ro1rjg" target="_blank" div class="certificate">
```
Comment (YauheniBeiduk) [inline index.html:54]: also here

File: index.html [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+            </div>
+            <div class="about indent">&#10022;&ensp;ABOUT ME
+                <p>&#8658; I want to join a team that don't know the word stop, in which I can develop.</p>
+                <p>&#8658; I good plan my working hours. I can understand else's developers code, I have an analytical
+                    mind.
+                    Read the book "Stoyan Stefanov - JavaScript Shablony (2011)".
+                    Cheerful and sociable, I always find a common language with the new team.
+                </p>
+            </div>
+            <div class="certificates indent">&#10022;&ensp;CERTIFICATES
```
Comment (YauheniBeiduk) [inline index.html]: Why here do you use `div` and in another cases with tag `a` you use `p`??

File: index.html [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+        <section class="content">
+            <div class="imformation indent">&#128190;&ensp;INFORMATION
+                <p>I believe my greatest strength is my ability to solve problems quickly and efficiently. I can see any
+                    situation from different angles, which gives me a unique qualification to do my job even in
+                    difficult conditions.</p>
+            </div>
+            <div class="skill indent">&#9989;&ensp;SKILLS
+                <div class="container_skills">
+                    <div class="skills html"><span>HTML</span> 90%</div>
+                </div>
```
Comment (YauheniBeiduk) [inline index.html]: I think we can remove one `div`

File: index.html [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+                <p>EPAM Stydied Front-end developer (Up Skill)</p>
+                <p>09/2020 - 01/2021</p>
+            </div>
+            <div class="project indent">&#128293;&ensp;PET-PROJECT</div>
+            <div class="container_sidebar">
+                <div class="sidebar">
+                    <div style="background: linear-gradient(229.99deg,
+                    #715c5e -26%, 
+                    #d000da 145%);">
+                        <h1><a href="https://rslang-team38-uladzislaukutarkin.netlify.app/"
```
Comment (YauheniBeiduk) [inline index.html:80]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

We use one h1 tag

File: index.html [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+                                style="text-decoration: none; color: white; font-size: 3vh;">LangApp</a></h1>
+                        <p>React</p>
+                    </div>
+                    <div style="background: linear-gradient(215.32deg, 
+                    #37b4d6 -7%, 
+                    #349552 124%);">
+                        <h1><a href="https://alekseygrimm.github.io/board-color/"
+                                style="text-decoration: none; color: white; font-size: 3vh;">BorderColor</a></h1>
```
Comment (YauheniBeiduk) [inline index.html:88]: also here

File: index.html [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+                        <p>HTML, CSS, JS</p>
+                    </div>
+                    <div style="background: linear-gradient(221.87deg, 
+                    #526259 40%, 
+                    #6e555d);">
+                        <h1><a href="http://alekseygrimm.github.io/Fancy-weather"
```
Comment (YauheniBeiduk) [inline index.html:94]: also here

File: index.html [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+                                style="text-decoration: none; color: white; font-size: 3vh;">Weather</a></h1>
+                        <p>Webpack</p>
+                    </div>
+                    <div style="background: linear-gradient(220.16deg, 
+                    #090f1d -8%, 
+                    #b4c7e3 138%);">
+                        <h1><a href="https://uladzislaukutarkin.github.io/travel-app/#/"
```
Comment (YauheniBeiduk) [inline index.html:101]: also here

File: index.html [hunk 12]
```diff
@@ -0,0 +1,138 @@
...
+                        <p>React</p>
+                    </div>
+                    <div style="background: linear-gradient(215.32deg, 
+                    #37b4d6 -7%, 
+                    #349552 124%);">
+                        <h1><a href="https://alekseygrimm.github.io/board-color/"
+                                style="text-decoration: none; color: white; font-size: 3vh;">BorderColor</a></h1>
+                        <p>HTML, CSS, JS</p>
+                    </div>
+                    <div style="background: linear-gradient(221.87deg,
```
Comment (YauheniBeiduk) [inline index.html:91]: Why do you use inline styles here? move it to styles.css

File: index.html [hunk 13]
```diff
@@ -0,0 +1,138 @@
...
+                        <h1><a href="https://uladzislaukutarkin.github.io/travel-app/#/"
+                                style="text-decoration: none; color: white; font-size: 3vh;">TravelApp</a></h1>
+                        <p>React</p>
+                    </div>
+                </div>
+                <div class="main-slide">
+                    <div style="background-image: url('./img/travelApp.png');"></div>
+                    <div style="background-image: url('./img/Weather.png');"></div>
+                    <div style="background-image: url('./img/Border-random-color.png');"></div>
+                    <div style="background-image: url('./img/RsLang.png');"></div>
```
Comment (YauheniBeiduk) [inline index.html:110]: Why do you use inline styles here? move it to styles.css

File: styles.css [hunk 14]
```diff
@@ -0,0 +1,316 @@
...
+    font-family: "Helvetica";
+    src: url("./font/HelveticaRegular.woff") format("woff");
+    font-weight: normal;
+    font-style: normal;
+    font-display: swap;
+    }
+
+html{
+    display: flex;
+    justify-content: center;
```
Comment (YauheniBeiduk) [inline styles.css:11]: I think we need to add flex to body, not to html

File: styles.css [hunk 15]
```diff
@@ -0,0 +1,316 @@
...
+}
+
+.container .controls .up-button {
+    transform: translateY(-100%);
+    border-top-right-radius: 5px;
+    border-bottom-right-radius: 5px;
+}
+
+@media screen and (min-width: 400px) and (max-width: 680px){
+
```
Comment (YauheniBeiduk) [inline styles.css:285]: you need to use breakpoints (320-768, 768-992, 992+)
https://www.w3schools.com/css/css_rwd_mediaqueries.asp

File: index.html (hunk 1 above)
Comment (AlekseyGrimm) [reply index.html:14]: done

File: index.html (hunk 2 above)
Comment (AlekseyGrimm) [reply index.html:24]: I used generic CSS properties for sections

File: index.html (hunk 3 above)
Comment (AlekseyGrimm) [reply index.html]: done

File: index.html (hunk 4 above)
Comment (AlekseyGrimm) [reply index.html:51]: I'm fixed it

File: index.html (hunk 5 above)
Comment (AlekseyGrimm) [reply index.html:54]: update

File: index.html (hunk 6 above)
Comment (AlekseyGrimm) [reply index.html]: I'm updated this code

File: index.html (hunk 7 above)
Comment (AlekseyGrimm) [reply index.html]: updeted this code

File: index.html (hunk 8 above)
Comment (AlekseyGrimm) [reply index.html:80]: I wanted exactly this size

File: index.html (hunk 9 above)
Comment (AlekseyGrimm) [reply index.html:88]: I wanted exactly this size

File: index.html (hunk 10 above)
Comment (AlekseyGrimm) [reply index.html:94]: I wanted exactly this size

File: index.html (hunk 11 above)
Comment (AlekseyGrimm) [reply index.html:101]: I wanted exactly this size

File: index.html (hunk 12 above)
Comment (AlekseyGrimm) [reply index.html:91]: https://github.com/Front-Grodno/Front-Grodno-JS-course-2021/tree/master/modules/css#practice
p.s. Use attributes (class, id, tag name)

Comment (AlekseyGrimm) [reply index.html:91]: I wanted to see how it works

File: index.html (hunk 1 above)
Comment (YauheniBeiduk) [reply index.html:14]: `section-wrapper_first`

**Pull request merged**